example/draw_lines.rb

```diff
@@ -18,11 +18,11 @@
 starting = Process.clock_gettime(Process::CLOCK_MONOTONIC)
 
 lines = image
-lines = lines.mutate do |x|
-  (0 .. 1).step 0.01 do |i|
+lines = lines.mutate { |x|
+  (0..1).step 0.01 do |i|
```

Comment:
This one feels odd to me. Multiline `{ }` look ugly, and it's not a functional block.

Reply:
yep, this somehow become popular to use `{}` for blocks with returns

Reply:
not popular, but has been preselected by authors of the standard. They prefer to have semantics blocks

Reply:
disabled semantic blocks